### PR TITLE
Merge 8.1.x into develop 02/07/2024

### PR DESCRIPTION
**Pull request by matks**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Merge 8.1.x into develop 02/07/2024
| Type?             | improvement
| Category?         | ME
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | UI tests should be green
| UI Tests          |  https://github.com/matks/ga.tests.ui.pr/actions/runs/9757253162
| Fixed issue or discussion?     | 
| Related PRs       | 
| Sponsor company   | PrestaShop SA

## Git merge

```
Changes to be committed:
	modified:   classes/Dispatcher.php
	modified:   classes/controller/FrontController.php
	modified:   docs/CHANGELOG.txt
	modified:   js/admin.js

Conflicts:
	both modified:   CONTRIBUTORS.md
	deleted on develop:   admin-dev/themes/default/template/controllers/statuses/helpers/form/form.tpl
```

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -249,7 +251,10 @@ GitHub contributors:
 - DogSports
 - Dominic Taylor
 - Dominik Shaim Ulrich
+<<<<<<< HEAD
 - dparis
+=======
+>>>>>>> 8.1.x
```
Comment (boherm) [inline CONTRIBUTORS.md]: You have a conflict that remains here!

**Review by boherm — CHANGES_REQUESTED** [review]

File: CONTRIBUTORS.md (hunk 1 above)
Comment (matks) [reply CONTRIBUTORS.md]: Nice catch, fixed here https://github.com/PrestaShop/PrestaShop/pull/36475/commits/48bf66387cf7394ca6738b7c5f73227b67f7867a

**Review by nicosomb — CHANGES_REQUESTED** [review]
You have to launch UI tests on MariaDB too.

**Comment by matks** [comment]
> You have to launch UI tests on MariaDB too.

We did not agree to force everyone to launch UI tests on both SQL engines 🤔

**Review by boherm — APPROVED** [review]

**Review by nicosomb — APPROVED** [review]

**Review by florine2623 — APPROVED** [review]
CI is green, no need functional tests from QA

**Pull request merged**